Sources/MongoSwift/MongoCollection.swift

```diff
@@ -12,6 +12,21 @@ public struct DropCollectionOptions: Codable {
     }
 }
 
+/// Options to use when renaming a collection.
+public struct RenamedCollectionOptions: Codable {
```

Comment:
I think this makes more sense as `RenameCollectionOptions` or `RenameOptions` since they're configuring the underlying command, whereas the past tense is mostly for communicating the semantics of the method invocation. What do you think @kmahar ?